.github/workflows/lighthouse.yml

```diff
@@ -12,32 +12,35 @@ on:
 
 # A workflow run is made up of one or more jobs that can run sequentially or in parallel
 jobs:
-  # This workflow contains a single job called "build"
-  build:
+  # This workflow contains a single job
+  lighthouse:
     # The type of runner that the job will run on
     runs-on: ubuntu-latest
 
     # Steps represent a sequence of tasks that will be executed as part of the job
     steps:
     # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
     - uses: actions/checkout@v2
+    - name: Setup Node
+      uses: actions/setup-node@v1
 
-    # Runs a single command using the runners shell
-    - name: Run a one-line script
-      run: echo Hello, world!
-
-    # Runs a set of commands using the runners shell
-    - name: Run a multi-line script
+    - name: NPM install and Build
       run: |
-        echo Add other actions to build,
-        echo test, and deploy your project.
+        npm i
+        npm run build
+
+    - name: Waiting for 200 from the Netlify Preview
+      uses: jakepartusch/wait-for-netlify-action@v1
+      id: waitFor200
+      with:
+        site_name: "happy-franklin-69b6d4"
```

Comment:
Great. I see that you are waiting for the netlify preview to deploy before running lighthouse tests.

Reply:
yes, reached it finally 😆